Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Build the image in a distinct stage so we don't need the Golang SDK.
+FROM golang:1.11-stretch as builder
+
+# Install pre-reqs.
+#     - Update apt-get sources
+RUN apt-get update -y
+#     - Dep, for Go package management
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+
+# Copy the source code over, restore dependencies, and get ready to build everything. We copy the Gopkg
+# files explicitly first to avoid excessive rebuild times when dependencies did not change.
+WORKDIR /go/src/github.com/pulumi/pulumi
+COPY Gopkg.* ./
+RUN dep ensure -v --vendor-only
+COPY . .
+
+# Build the CLI itself.
+RUN make install
+
+# Build the plguins for each of the language hosts (Go, Node.js, Python). This just builds Go code, and
+# specifically avoids building the packages, since those are distributed using package managers.
+RUN cd sdk/go && make install_plugin
+RUN cd sdk/nodejs && make install_plugin
+RUN cd sdk/python && make install_plugin
+
+# Install and run in Alpine Linux.
```

Comment:
This comment doesn’t look right given current approach.

Reply:
Ah good catch. I started by trying to get Alpine to work, but ran into too many issues. Will fix